### DISCUSSION TITLE
Can this use browser local storage as a backend?

### DISCUSSION THREAD

**Discussion by radiantone** [n1crack/vuefinder#58] [General]
For example, a browser local storage database as the storage option

**Comment by n1crack** [comment]
Yes, its possible now. We have drivers now. (LocalDriver and RemoteDriver). 
you can also build a custom one and pass it as a param.

**Reply by radiantone** [reply]
> Yes, its possible now. We have drivers now. (LocalDriver and RemoteDriver). you can also build a custom one and pass it as a param.

That's amazing!!!! Thank you so much.

**Reply by n1crack** [reply]
Still it may have issues, but in time we can solve things. Many things changed recently.

**Reply by radiantone** [reply]
> > Yes, its possible now. We have drivers now. (LocalDriver and RemoteDriver). you can also build a custom one and pass it as a param.
> 
> That's amazing!!!! Thank you so much.

What storage option do I use for LocalDriver so it uses browser database? I see only 'memory'

**Reply by n1crack** [reply]
Ah, it uses simply an array ref( as memory)

What storage do you need ? I can look for it.

**Reply by radiantone** [reply]
> Ah, it uses simply an array ref( as memory)
> 
> What storage do you need ? I can look for it.

IndexedDB API - Web APIs | MDN https://share.google/UGO5NkGs1pqLO4ylh

**Reply by n1crack** [reply]
Nice, I'll try to create a driver for IndexDb.

**Reply by n1crack** [reply]
renamed LocalDriver as ArrayDriver
added a IndexedDbDriver..  and a basic example.. 

it has issues, I'll look into it when I have more time.
